Documentation/Books/Manual/Views/ArangoSearch/Scorers.md

```diff
@@ -1,58 +1,47 @@
 ArangoSearch Scorers
 ====================
 
-ArangoSearch accesses scorers directly by their internal names. The
-name (in upper-case) of the scorer is the function name to be used in the
-['SORT' section](../../../AQL/Views/ArangoSearch/index.html#arangosearch-sort).
-Function arguments, (excluding the first argument), are serialized as a
-string representation of a JSON array and passed directly to the corresponding
-scorer. The first argument to any scorer function is the reference to the 
-current document emitted by the `FOR` statement, i.e. it would be 'doc' for this
-statement:
+ArangoSearch Scorers are special functions that allow to sort documents from a
+view by their score regarding the analyzed fields.
 
-```js
-FOR doc IN someView
-```
+Details about their usage in AQL can be found in the [ArangoSearch `SORT`
+section](../../../AQL/Views/ArangoSearch/index.html#arangosearch-sort).
 
-IResearch provides a 'bm25' scorer implementing the
-[BM25 algorithm](https://en.wikipedia.org/wiki/Okapi_BM25). This scorer
-optionally takes 'k' and 'b' positional parameters.
-
-The user is able to run queries with the said scorer, e.g.
-
-```js
-SORT BM25(doc, 1.2, 0.75)
-```
-
-The function arguments will then be serialized into a JSON representation:
+- BM25: order results based on the [BM25 algorithm](https://en.wikipedia.org/wiki/Okapi_BM25)
 
-```json
-[ 1.2, 0.75 ]
-```
+- TFIDF: order results based on the [TFIDF algorithm](https://en.wikipedia.org/wiki/TF-IDF)
 
-and passed to the scorer implementation.
+### `BM25()` - Best Matching 25 Algorithm
 
-Similarly an administrator may choose to deploy a custom DNA analyzer 'DnaRank'.
+IResearch provides a 'bm25' scorer implementing the [BM25
+algorithm](https://en.wikipedia.org/wiki/Okapi_BM25).  Optionally, the term
+frequency **k** and coefficient **b** of the algorithm can be specified as
+floating point numbers.
 
-The user is then immediately able to run queries with the said scorer, e.g.
+`BM25(doc, k, b)`
 
-```js
-SORT DNARANK(doc, 123, 456, "abc", { "def": "ghi" })
-```
+- *doc* (document): must be emitted by `FOR doc IN someView`
 
-The function arguments will then be serialized into a JSON representation:
+- *k* (number, _optional_): term frequency, the default is _1.2_.  *k*
```

Comment:
k and b are free parameters typically using for advanced optimization. There is nothing related to "term frequency"

Reply:
Fixed.